### ISSUE TITLE
Object is not subscriptable - Error

### ISSUE THREAD

**Issue by PJ1004** [ryanharrison554/pyovpn-as#2]
Unable to use this module - receive an error when attempting to import the pip module within a python script

To replicate:

### Dockerfile

```
FROM python:3.8
RUN pip install --upgrade pip && \
    pip install pyovpn_as
CMD ["/bin/bash"]
```

### Test

```
python
>>> from pyovpn_as import client
```

### Output

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/pyovpn_as/client.py", line 9, in <module>
    from . import utils
  File "/usr/local/lib/python3.8/site-packages/pyovpn_as/utils.py", line 9, in <module>
    from pyovpn_as.api.cli import RemoteSacli
  File "/usr/local/lib/python3.8/site-packages/pyovpn_as/api/cli.py", line 37, in <module>
    class RemoteSacli:
  File "/usr/local/lib/python3.8/site-packages/pyovpn_as/api/cli.py", line 157, in RemoteSacli
    ) -> list[bool, dict]:
TypeError: 'type' object is not subscriptable
```

**Comment by peritz** [comment]
Fixed with #3, I'll push the next version now

**Comment by PJ1004** [comment]
Thanks for sorting that - can now load the project in python.
